src/game/g_local.h

```diff
@@ -490,11 +472,10 @@ extern  int snd_fry;
 #define MOD_TRIGGER_HURT    31
 #define MOD_HIT             32
 #define MOD_TARGET_BLASTER  33
-#define MOD_FRIENDLY_FIRE   0x8000000
```

Comment:
`0x8000000 != BIT(31)` (one 0 too short)
which one is correct?

Reply:
The value in the original game sources is "one 0 too short".
However, I don't think the actual value really matters: AFAICS, the "means of death" isn't actually something that's persisted, so `MOD_FRIENDLY_FIRE` can be any flag value, as long as it doesn't conflict with any other `MOD_xxx` value - which is the case for either value.
(I suspect that, when this change was made in Q2PRO, either `0x8000000` was misread to contain an extra zero, or it was decided that it's "nicer" to use the MSB as the flag bit, hence the change to `BIT(31)`.)